src/Widgets/TextWidget/TextWidgetComponent.js

```diff
@@ -1,31 +1,12 @@
 import * as React from "react";
 import * as Scrivito from "scrivito";
-
-Scrivito.provideComponent("TextWidget", ({ widget }) => {
-  const classNames = [alignmentClassName(widget.get("alignment"))];
-
-  return (
-    <Scrivito.ContentTag
-      tag="div"
-      className={classNames.join(" ")}
-      content={widget}
-      attribute="text"
-    />
-  );
-});
-
-function alignmentClassName(widgetAlignment) {
-  if (widgetAlignment === "left") {
-    return "text-start";
-  }
-
-  if (widgetAlignment === "center") {
-    return "text-center";
-  }
-
-  if (widgetAlignment === "right") {
-    return "text-end";
-  }
-
-  return "";
-}
+import { alignmentClassName } from "../../utils/alignmentClassName";
+
+Scrivito.provideComponent("TextWidget", ({ widget }) => (
+  <Scrivito.ContentTag
+    tag="div"
+    className={alignmentClassName(widget.get("alignment"))}
```

Comment:
* [ ] FYI In contrast to passing it to `WrapIfClassName` this usage of `alignmentClassName` will lead to a `class=""` attribute. We could make the behavior more consistent (in the sense of "the default doesn't render unnecessary stuff") if `alignmentClassName` returned `undefined`/`null`/false.